### DISCUSSION TITLE
What do you think about the activity / project backlogs?

### DISCUSSION THREAD

**Discussion by choldgraf** [2i2c-org/team-compass#79] [General]
If you've got any thoughts about the activity and project backlogs, or any questions about them, feel free to put them here so that we can discuss!

**Comment by damianavila** [comment]
For the development one (deliverables), I think that one is screaming for filtering by prioritization (high, medium, low) and major project labels (Managed Hub, Executable Book, notebook sharing) if that is possible...

**Reply by choldgraf** [reply]
@damianavila two quick thoughts below:
1. For Prioritization, what do you think about adding these labels across all of our repositories:

    - `prio: low`, `prio: med`, `prio: high`
    - And removing the `:fire: priority` label
2. For the project labels, how do you think we could do this, given that some of the projects are in other organizations? Two ideas:
    - Create a label in all of the EBP repositories that we add to all issues we want to work on (though, this would seem weird to others in the EBP community, I suspect)
    - Create an issue to track major deliverables that *we* want to work on in a 2i2c repository (e.g., `2i2c-org/development`?). These issues would just be cross-links to the respective EBP repository, but this could be labelled/tagged however we like.

**Reply by damianavila** [reply]
> For Prioritization, what do you think about adding these labels across all of our repositories:

I like that

>  given that some of the projects are in other organizations?

Yep, I realized that could be a problem...

> Create a label in all of the EBP repositories that we add to all issues we want to work on (though, this would seem weird to others in the EBP community, I suspect)

As you said, it is weird and confusing for others in that community.

> Create an issue to track major deliverables that we want to work on in a 2i2c repository (e.g., 2i2c-org/development?). These issues would just be cross-links to the respective EBP repository, but this could be labelled/tagged however we like.

Even when that is a dupe of info, I think is probably the way to go... in fact, that is somehow our (2i2c) view of our contributions and prioritization in the EBP project, that could be aligned but not always in sync with that whole community, so kind of makes sense, I think.

**Reply by choldgraf** [reply]
@damianavila thinking about this a bit more - how about we create a repository called `2i2c-org/projects`. This would act as a "meta" repository that tracks the different major projects that we have going on.  It would be the Source of Truth for:

- the list of "active major projects at 2i2c"
- answering "where are the deliverables tracked for each project?"
- deliverables issues that do not have a natural home inside a `2i2c-org` repository (e.g. executablebooks issues). Each of these issues would have a `proj: <projectname>` label

Then, we would re-direct [this page in the team compass](https://team-compass.2i2c.org/en/latest/reference/projects.html) to point to that repository.

**Reply by damianavila** [reply]
I like that!
